starter_code/src/movies.js

```diff
@@ -1,6 +1,19 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(duration) {
+    var splitDuration = duration.split("h ");
```

Comment:
In the data.js file, you can see that sometimes the duration is expressed only in minutes, like "22min" or only in hours, like "2h". Your code will break for those test cases